boot/zephyr/main.c

```diff
@@ -148,6 +148,25 @@ K_SEM_DEFINE(boot_log_sem, 1, 1);
 #include <nrf_cleanup.h>
 #endif
 
+#include <zephyr/linker/linker-defs.h>
+#define CLEANUP_RAM_GAP_START ((int)__ramfunc_region_start)
+#define CLEANUP_RAM_GAP_SIZE ((int) (__ramfunc_end - __ramfunc_region_start))
```

Comment:
It would be significantly easier to manage if we could relocate the function/section to the end of SRAM. Then would not have to play with the "gap"